cogs/cog_easteregg.py

```diff
@@ -0,0 +1,218 @@
+                url="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.istockphoto.com%2" +
+                    "Fphotos%2Frock-paper-scissors-picture-id172245258%3Fk%3D6%26m%3D172245258%26s%3D612x6" +
+                    "12%26w%3D0%26h%3DNh1HajvkB5bANhHZ6Ny9w9tNleYVaBwp7jwnNdEtjPg%3D&f=1&nofb=1")
+            msg = await ctx.send(embed=embed)
+
+            for emoji in emojis:  # add emoji to the message
+                await msg.add_reaction(emoji)
+
+            # get user input
+            reaction, user = await self.bot.wait_for('reaction_add',
+                                                     timeout=60.0,
+                                                     check=(lambda emj, usr:
+                                                            usr == ctx.author
+                                                            and emj.message == msg
+                                                            and emj.emoji in emojis))
+
+            bot_choice = random.choice(['Rock', 'Paper', 'Scissors'])
+
+            # set emoji and image for bot
+            if bot_choice == "Rock":
+                emoji = emojis[0]
+                img = "https://jooinn.com/images/rock-1.jpg"
+            elif bot_choice == "Paper":
+                emoji = emojis[1]
+                img = "https://i.ytimg.com/vi/hFFpoIv7YKg/maxresdefault.jpg"
+            else:
+                emoji = emojis[2]
+                img = "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia." \
+                      "istockphoto.com%2Fphotos%2Fopen-scissors-with-red-handle-isolated" \
+                      "-on-white-picture-id517231506%3Fk%3D6%26m%3D517231506%26s%3D612x6" \
+                      "12%26w%3D0%26h%3DFG8dBeuKmNzA44ee-lFMdKzlLYxGnYcFVTMaIbHvNeE%3D&f=1&nofb=1"
+
+            # win-lose-tie casses
+            if (bot_choice == "Rock" and reaction.emoji == emojis[0])\
+                    or (bot_choice == "Paper" and reaction.emoji == emojis[1])\
+                    or (bot_choice == "Scissors" and reaction.emoji == emojis[2]):  # tied
+                color = cfg.colors.LINK
+                title = "Tied!"
+            elif (bot_choice == "Paper" and reaction.emoji == emojis[0])\
+                    or (bot_choice == "Scissors" and reaction.emoji == emojis[1])\
+                    or (bot_choice == "Rock" and reaction.emoji == emojis[2]):  # bot wins
+                color = cfg.colors.ERROR
+                title = "Oh no! you lost."
+            else:  # player wins
+                color = cfg.colors.SUCCESS
+                title = "Yay! you won."
+
+            # output embed
+            embed = discord.Embed(title=title,
+                                  description=f"Bot: {emoji}\nYou: {reaction.emoji}",
+                                  color=color)
+            embed.set_author(name=ctx.author.display_name, icon_url=ctx.author.avatar_url)
+            embed.set_thumbnail(
+                url=img)
```

Comment:
Is the variable title in line 128 the same variable being assigned in line 131 to title? Having a hard time differentiating them.

Reply:
Yes they are, I will change the assigned title to custom_title